res/css/views/rooms/_MessageComposer.scss

```diff
@@ -63,6 +63,22 @@ limitations under the License.
     .mx_VoiceMessagePrimaryContainer {
         margin-right: $spacing-8;
     }
+
+    .mx_ReplyPreview_header {
```

Comment:
That code should be in `_ReplyPreview.scss` most probably

Reply:
I am not sure because it styles an element on the message composer in thened. Moving it to `_ReplyPreview` has global effect, which I don't think is expected here.

Reply:
The `ReplyPreview` component is only used in message composer. I think it's safe to put it there

Reply:
Hm... it's your call.